Source/Python/pyWarpX.cpp

```diff
@@ -71,8 +71,8 @@ PYBIND11_MODULE(PYWARPX_MODULE_NAME, m) {
 
     // API runtime version
     //   note PEP-440 syntax: x.y.zaN but x.y.z.devN
-#ifdef PYIMPACTX_VERSION_INFO
-    m.attr("__version__") = MACRO_STRINGIFY(PYIMPACTX_VERSION_INFO);
+#ifdef pyWarpX_VERSION_INFO
```

Comment:
Should this be all caps?

Reply:
Yes, but gotta change it in the CMake file as well.

Reply:
Good catch!